config/config.yaml

```diff
@@ -0,0 +1,44 @@
+internalApi:
+  port: 8081
+  gracefulShutdownTimeout: 10s
+  metrics:
+    enabled: true
+  healthcheck:
+    enabled: true
+api:
+  port: 8080
+  gracefulShutdownTimeout: 10s
+
+postgres:
+  url: "postgres://maestro:maestro@localhost:5432/maestro?sslmode=disable"
+redis:
+  url: "redis://localhost:6379/0"
```

Comment:
If we were to start the project, I would totally go this way, having only one redis-url and postgres-url. However, since we already have one redis for each adapter, I think we could keep it, since it gives us the possibility to have one redis for each adapter. What do you think about keeping the adapters format we had before but in this unified config file?

Reply:
Totally agree with @caiordjesus, i think that the greatest pain that we currently have is to need to define redis URLs for each maestro component (and this PR already fixes it), and it doesn't make sense to have different maestro components pointing to different redis, but it does make sense to be able to have different redis for different ports, (e.g one redis for game rooms and another just for operations)

Reply:
PS: Not a blocking change, consider it just if doesn't take too much effort